pages/index.vue

```diff
@@ -1,9 +1,19 @@
 <template>
   <div>
+    <div class="small-banner">
```

Comment:
The order of the cms components for this page should render in the same order it is configured in on the cms.

Reply:
currently, we do not have separate components similar to Next JS. I think we should create separate ticket for this refactoring.